pkg/configmap/configmap_watcher.go

```diff
@@ -1,8 +1,9 @@
-package controller
+package configmap
```

Comment:
Wouldn't be better to call the package just `config` ?

Reply:
I had a lint complaint (fixed already) to rename the structure inside the file as Watcher, instead of ConfigmapWatcher.

I think `configmap.Watcher` is way more specific and readable than`config.Watcher`.

Also, there is some other packages imported inside the project with the generic name `config`.

But I don't have strong opinion on this, if you think it's better to call it config, it's a minimal effortless change.

Reply:
OK. Yes `configmap.Watcher` is probably better. It was just a suggestion...you can keep it as it is.